deploy/010_serviceaccount-rolebinding.yaml

```diff
@@ -5,16 +5,40 @@ metadata:
   name: sre-stuck-ebs-vols
   namespace: openshift-monitoring
 ---
-apiVersion: authorization.openshift.io/v1
+apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
   name: sre-stuck-ebs-vols
   namespace: openshift-monitoring
 roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
   name: edit
 subjects:
 - kind: ServiceAccount
   name: sre-stuck-ebs-vols
   namespace: openshift-monitoring
-userNames:
-- system:serviceaccount:openshift-monitoring:sre-stuck-ebs-vols
+---
+kind: ClusterRole
```

Comment:
Does this need to be a `ClusterRole`?

Reply:
@jewzaam Yes, because the object is in a different namespace.

`Role` are for just one namespace but `ClusterRole` are for all namespaces. (We narrow the scope from "all namespaces" to `openshift-machine-api` with the following `RoleBinding`)